utils/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -62,13 +62,22 @@ object DeclaredLicenseProcessor {
         }
     }
 
+    /**
+     * Return a non-null value if the given [declaredLicense] could be turned into an [SpdxExpression] utilizing
+     * internal mapping data.
+     */
     internal fun process(declaredLicense: String): SpdxExpression? {
         val licenseWithoutPrefixOrSuffix = preprocess(declaredLicense)
         val mappedLicense = SpdxDeclaredLicenseMapping.map(licenseWithoutPrefixOrSuffix)
 
         return (mappedLicense ?: parseLicense(licenseWithoutPrefixOrSuffix))?.normalize()?.takeIf { it.isValid() }
     }
 
+    /**
+     * Return [ProcessedDeclaredLicense] derived from the given [declaredLicenses]. The processing tries to map each
+     * given declared license to an [SpdxExpression] by first applying a removal step for known URL prefixes (and
```

Comment:
typo: "license**s**"

Reply:
I believe it is correct because of the preceeding `each`, is it?